src/app/modules/inbox/inbox.component.ts

```diff
@@ -189,7 +189,7 @@ export class InboxComponent implements OnInit, OnDestroy {
                         'page': 1,
                         'tag_id': this.emailParentId,
                         'default_id': this.emailChildId,
-                        'limit': 100,
+                        'limit': 20,
```

Comment:
since this is used at many places, make this a constant in config file

Reply:
okay sir.